src/consensus/validation.h

```diff
@@ -12,24 +12,17 @@
 #include <primitives/transaction.h>
 #include <primitives/block.h>
 
-/** "reject" message codes */
-static const unsigned char REJECT_MALFORMED = 0x01;
+/** A reject code must be provided for P2P reject messages. Just always send REJECT_INVALID
+ *  TODO: remove support for BIP 61 reject messages.
```

Comment:
nit: `*` is not aligned quite right

```suggestion
  * TODO: remove support for BIP 61 reject messages.
```

Reply:
thanks. Fixed!